src/fidesops/schemas/analytics.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class EVENT(Enum):
```

Comment:
Maintaining the event names like this is a fantastic idea!! This is definitely something that should be done in fidesctl too.

Just one suggestion: since all of the event names will likely be strings:
```suggestion
class EVENT(str, Enum):
```
This will enable some nice shortcuts if we ever need to manipulate or compare the names  outside of this `Enum` class.